esignet-core/src/main/java/io/mosip/esignet/core/validator/ClaimSchemaValidator.java

```diff
@@ -0,0 +1,86 @@
+package io.mosip.esignet.core.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationMessage;
+import io.mosip.esignet.api.dto.claim.ClaimsV2;
+import io.mosip.esignet.core.exception.EsignetException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.web.client.RestTemplate;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+@Slf4j
+public class ClaimSchemaValidator implements ConstraintValidator<ClaimSchema, ClaimsV2> {
+
+
+    @Value("${mosip.esignet.json.validation.schema.url}")
```

Comment:
Rename the property as "mosip.esignet.claims.schema.url"

Reply:
changed